features/staging_commands.feature

```diff
@@ -46,6 +48,32 @@ Feature: Enable command behaviour when attached to an UA staging subscription
         \s*\*\*\* .* 500
         \s*500 https://esm.staging.ubuntu.com/apps/ubuntu <release>-apps-security/main amd64 Packages
         """
+        When I run `mkdir -p /var/lib/ubuntu-advantage/messages` with sudo
+        When I run `sh -c 'echo "esm-apps {ESM_APPS_PKG_COUNT} {ESM_APPS_PACKAGES}; esm-infra {ESM_INFRA_PKG_COUNT} {ESM_INFRA_PACKAGES}" > /var/lib/ubuntu-advantage/messages/contract-expired-apt-upgrade.tmpl'` with sudo
+        When I run `/usr/lib/ubuntu-advantage/apt-esm-hook process-templates` with sudo
+        When I run `cat /var/lib/ubuntu-advantage/messages/contract-expired-apt-upgrade` with sudo
```

Comment:
This command is failing, which presumably means the apt-hook line is failing to output the file, but I can't recreate in manual testing.

Reply:
After rebooting my machine this error disappeared